runeq/resources/project.py

```diff
@@ -145,13 +145,26 @@ def get_patient_metadata_dataframe(self) -> pd.DataFrame:
         Returns a new dataframe displaying the patient's metadata.
 
         """
+        column_name_map = {
+            "type": "metric_type",
+            "data_type": "metric_data_type",
+            "value": "metric_value",
+            "time_interval": "metric_time_interval",
+            "created_at": "metric_created_at",
+            "updated_at": "metric_updated_at",
+            "id": "metric_id",
+        }
         metadata_obj = self.to_dict()
-        metadata_df = pd.DataFrame(metadata_obj)
+        metadata_df = pd.DataFrame(metadata_obj).rename(
+            columns={"code_name": "project_code_name"}
```

Comment:
Just double checking here, the screenshot in the PR notes has values, for the `project_code_name` column, that are prefixed `patient-code-name-`. Is that supposed to be the case, or should the column be named `patient_code_name`?

Reply:
So the code names in this column are project specific, so it is a patient code name, but it is specific to the project (project code name). The screen shot just had the unfortunate coincidence of the project patient code name being `patient-code-name-114562`

Reply:
does that make sense?

Reply:
I could also label the column `project patient code name` to make it clear

Reply:
Ah got it, it's the patient's code name within the project. Yeah I think `project_patient_code name` would be better, but don't have strong feelings.